server/routes/makeAReferral/makeAReferralController.ts

```diff
@@ -797,4 +849,23 @@ export default class MakeAReferralController {
 
     ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
+
+  async generateCaseNote(accessToken: string, referralId: string, subject: string, message: string): Promise<void> {
+    const caseNote = {
+      referralId,
+      subject,
+      body: message,
+    }
+    await this.interventionsService.addCaseNotes(accessToken, caseNote)
+  }
+
+  async isSentReferral(req: Request, res: Response): Promise<boolean> {
+    try {
```

Comment:
we need to explicitly check for 404 here on the draft referral call. 

also - if the second call fails or also returns 404, what do we do?

Reply:
i would guess we probably fail in some way. do we need to fail now? can we assume the referral is sent and have code further down the chain handle the fact that something is wrong? that would save an additonal API call.